exporters/prometheus/CMakeLists.txt

```diff
@@ -0,0 +1,201 @@
+# Copyright 2020, OpenTelemetry Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+cmake_minimum_required(VERSION 3.12)
+
+project(
+  opentelemetry-cpp-contrib
+  VERSION "1.6.1" # opentelemetry-cpp-contrib 1.6.1
+  HOMEPAGE_URL "https://github.com/open-telemetry/opentelemetry-cpp-contrib"
+  LANGUAGES CXX)
+
+if(NOT CMAKE_ARCHIVE_OUTPUT_DIRECTORY)
+  set(CMAKE_ARCHIVE_OUTPUT_DIRECTORY
+      "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}")
+endif()
+
+if(NOT CMAKE_LIBRARY_OUTPUT_DIRECTORY)
+  set(CMAKE_LIBRARY_OUTPUT_DIRECTORY
+      "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR}")
+endif()
+
+if(NOT CMAKE_RUNTIME_OUTPUT_DIRECTORY)
+  set(CMAKE_RUNTIME_OUTPUT_DIRECTORY
+      "${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}")
+endif()
+
+link_directories(${CMAKE_ARCHIVE_OUTPUT_DIRECTORY})
+
+find_package(opentelemetry-cpp REQUIRED)
```

Comment:
No changes are required, but looking for a suggestion here. 
What should be the seamless way to install opentelemetry-cpp as part of such external exporter installation, to make life easier for the end user?
I tried something like - https://github.com/open-telemetry/opentelemetry-cpp-contrib/blob/c44186c708b16b1bd4a7ea8f47fa49d509015725/exporters/fluentd/CMakeLists.txt#L28 which uses `ExternalProject_Add` for fluentd exporter, but not sure if this is good enough. I feel we should sometime document an easy way to consume such external components.

Reply:
Good idea. I think we can always use `find_package(opentelemetry-cpp QUIET)` to find it first. And then use `ExternalProject_Add` when we can not find a prebuilt opentelemetry-cpp.
And also, maybe we can create alias for all cmake targets with the same names as exported names. Just like abseil-cpp, and then we can use the same name no matter we import opentelemetry-cpp by `find_package` or `add_subdirectory`?